CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+= 0.9.5 =
+
+Fixed:
+
+* Post-type filters intruduced in v0.9.4 now also affect the graph results.
+
+= 0.9.4 =
+
+Enhancements:
+
+* Added a setting to include post types, we default to `post` and `page` and you can add others as you wish.
```

Comment:
I see I forgot to set this from the readme, we should copy the text from the readme to here.

Reply:
What do you mean? I already copied it...